src/RoleObjects/AccessibilityObject.js

```diff
@@ -636,22 +754,20 @@ export default class AccessibilityObject {
 
   /**
    * Makes the DOM entry visible immediately.  This does not affect the value used during the next render of the accessibility tree.
-   * @access package
+   * @access public
    */
   _forceShow() {
     document.getElementById(this._domId).style.display = '';
   }
 
   /**
    * Finds the index into this.children for the specified DOM id
-   * @access package
+   * @access public
```

Comment:
please see previous comment

Reply:
@package similar to @access package
http://usejsdoc.org/tags-package.html#examples